src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1201,4 +1201,7 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="AddOverloadsKeyword" xml:space="preserve">
     <value>Add Overloads</value>
   </data>
+  <data name="ReferenceKeywordTooltip" xml:space="preserve">
+    <value>Loads a reference assembly</value>
```

Comment:
This looks different from C#'s: `Add a metadata reference to specified assembly and all its dependencies, e.g. #r "myLib.dll".`

Also, be careful with "reference assembly" since that's jargon in its own right.